src/app/modules/work-schedules/__mocks__/work-schedules-dependent-services.ts

```
@@ -0,0 +1,29 @@
+  paginate: jest.fn(),
+};
+
+export const mockOrganizationsService = {
+  createOrganiztion: jest.fn(),
+}; 
+ 
+ 
+ 
+ 
+ 
+import { OrganizationsService } from '../../organizations/organizations.service';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'OrganizationsService'. Consider to delete it or rename it.

'OrganizationsService' is defined here:


(lint/suspicious/noRedeclare)